### PR TITLE
fix issue #445 : https://github.com/influxdata/influxdb-java/issues/445

### PR DESCRIPTION
**Pull request by lxhoan**
Hi @majst01 , I accidentally dropped PR #477 so create this new one
PR is now mergable, please take a look, tks

### CODE REVIEW

File: src/test/java/org/influxdb/InfluxDBTest.java [hunk 1]
```diff
@@ -891,5 +895,63 @@ public void testMessagePackOnOldDbVersion() {
...
+    for (int i = 0; i < maxCallables; i++) {
+      callableList.add(new Callable<String>() {
+        @Override
+        public String call() throws Exception {
+          MyInfluxDBBean myBean = new MyInfluxDBBean();
+          return myBean.connectAndDoNothing1();
+        }
+      });
+    }
+    System.out.println("Invoking all callableList (size()=" + callableList.size() + ")");
```
Comment (majst01) [inline src/test/java/org/influxdb/InfluxDBTest.java]: I am not a big fan of having sysout in tests, are these necessary ?

File: src/test/java/org/influxdb/InfluxDBTest.java [hunk 2]
```diff
@@ -18,21 +36,7 @@
 import org.junit.platform.runner.JUnitPlatform;
 import org.junit.runner.RunWith;
 
-import java.io.IOException;
```
Comment (majst01) [inline src/test/java/org/influxdb/InfluxDBTest.java]: please revert this formatting

File: src/test/java/org/influxdb/InfluxDBTest.java (hunk 1 above)
Comment (lxhoan) [reply src/test/java/org/influxdb/InfluxDBTest.java]: removed them

File: src/test/java/org/influxdb/InfluxDBTest.java (hunk 2 above)
Comment (lxhoan) [reply src/test/java/org/influxdb/InfluxDBTest.java]: reverted as comments, thanks

**Pull request merged**